hdr_schemata/utils/create_markdown.py

```diff
@@ -219,6 +219,7 @@ def remove_types(data):
 
     form = {}
     form["schema_fields"] = []
+    form["url_regex"] = "(https?:\/\/)?([a-zA-Z0-9-]+\.)+[a-zA-Z]{2,}(:\d+)?(\/[^\s]*)?$"
```

Comment:
regex is forcing `https`, we'd want to allow `http` and maybe `www` ? Think the json-schema is a `uri` format that is accepted

Reply:
The s is optional so `http` or `https` both match. As does a url without the protocol e.g. `www.github.com`.

Reply:
You make a good point about the `uri` definition in the json-schema.  It allows port numbers among other things, so I've updated the regex to be more flexible. I also checked against url data we have in mk1 to make sure it validates against it too.